### ISSUE TITLE
AJAX Module does not exist on jsdelivr.net

### ISSUE THREAD

**Issue by 0xD3ADC0W** [brython-dev/brython#1771]
Code:

```python
  from browser import ajax

  def showShorten(req):
    if req.status == 200 or req.status == 0:
      alert(req.text)

  def shorten(event):

    req = ajax.Ajax()
    url = document["url"].value
    req.bind("complete", showShorten)
    req.open("POST" "/shorten", True)
    req.send({"url" : url})

  document["submit_button"].bind("click", shorten)
```

Causes this to be printed in the developer console.

```XHRGEThttps://cdn.jsdelivr.net/npm/brython@3.9.5/Lib/browser/ajax.py?v=1631818614447
[HTTP/3 404 Not Found 20ms]

Error 404 means that Python module browser.ajax was not found at url https://cdn.jsdelivr.net/npm/brython@3.9.5/Lib/browser/ajax.py
```

**Comment by 0xD3ADC0W** [comment]
Ok, so Brython by default is not importing the standard library. Adding this fixes the ajax module not being found. This isn't specified on the website:

```html
  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/brython@3.9.5/brython_stdlib.js"></script>
```

**Comment by PierreQuentel** [comment]
There is a typo in your code, you forgot a comma in `req.open("POST" "/shorten", True)`.

The commit referenced above prints an error message if the first 2 arguments of `open()` are not strings.

**Comment by 0xD3ADC0W** [comment]
> There is a typo in your code, you forgot a comma in `req.open("POST" "/shorten", True)`.
> 
> The commit referenced above prints an error message if the first 2 arguments of `open()` are not strings.

Just noticed, and like I said in my other issue I opened I will definitely start separating my HTML and python so I am less likely to make simple errors like this. As for the stdlib not being included by default, I think that should change.

I simply followed the [static tutorial](https://brython.info/static_tutorial/en/index.html) page before later poking around and noticing there was a standard library I didn't include. I shouldn't have skipped over the installation page assuming that it would just work from a CDN like JQuery.

There's no reference to the standard library there. The first reference I could find was under installation. I just assumed it would be batteries included (my bad), but I do think it should be mentioned on every single page as it's also not mentioned on many of the documentation pages such as [ajax](https://brython.info/static_doc/en/ajax.html) despite being a requirement to use the ajax module.

So I guess we could label this a suggestion? Anyways not trying to sound overly critical especially considering I'd rather walk on hot coals than build anything with the DOM. I do **extremely** appreciate all the work on Brython.
